packages/fxa-payments-server/src/components/CouponForm/index.tsx

```diff
@@ -187,23 +187,31 @@ export const CouponForm = ({
         </h4>
       )}
       {hasCoupon ? (
-        <div className="flex" data-testid="coupon-hascoupon">
-          <div className="mr-4">{promotionCode}</div>
+        <div
+          className="flex justify-between items-center"
+          data-testid="coupon-hascoupon"
+        >
+          <div>{promotionCode}</div>
           {readOnly ? null : (
-            <button
-              className={`button ${readOnly ? 'hidden' : ''}`}
-              onClick={removeCoupon}
-              disabled={subscriptionInProgress}
-              data-testid="coupon-remove-button"
-            >
-              <Localized id="coupon-remove">
-                <span>Remove</span>
-              </Localized>
-            </button>
+            <div className="w-28 ml-4">
```

Comment:
Per https://tailwindcss.com/docs/width#fixed-widths, setting `w-28`, sets `width: 7rem; /* 112px */`.
Is that risky if this is localized and some languages (ie: German) can be fairly "wordy"?

For giggles, I ran this in the fxa-content-server-l10n repo: <kbd>git grep -h "coupon-remove" | sort | uniq</kbd>
Then if I pipe it through some script which sorts by line length, we see that "Remove" is the 28th longest string (by character count, but not all character widths are equal, so it's a bit of a lame metric versus measured text width):

```sh
 1. coupon-remove = Eltávolítás
 2. coupon-remove = Verwijderen
 3. coupon-remove = Allontanar
 4. coupon-remove = Fuortsmite
 5. coupon-remove = Wotstronić
 6. coupon-remove = Wótwónoźeś
 7. coupon-remove = Премахване
 8. coupon-remove = Entfernen
 9. coupon-remove = Fjarlægja
10. coupon-remove = Odstrániť
…
28. coupon-remove = Remove
…
39. coupon-remove = 삭제
```

Where <kbd>git grep "Eltávolítás"</kbd> tells me that the string is Hungarian:
```
locale/hu/main.ftl:coupon-remove = Eltávolítás
```

If I hack that string into staging payments via DevTools it _just_ seems to fit, but adding 2-3 "m"s at the end causes it to start clipping text at both ends. But maybe it's a non-issue if we were hardcoding it to 110px earlier. 🤷 

![Screen Shot 2022-08-12 at 9 38 31 AM](https://user-images.githubusercontent.com/557895/184404322-7bd653bd-06ef-4b60-8ede-d08cf185db41.png)

Reply:
Excellent point! I removed the width restriction. I temporarily added extra padding to the sides as `button` and link elements will be addressed in FXA-5637.